RecoEgamma/EgammaTools/interface/SCEnergyCorrectorSemiParm.h

```diff
@@ -33,7 +33,8 @@ class SCEnergyCorrectorSemiParm {
   ~SCEnergyCorrectorSemiParm(); 
   
   void setTokens(const edm::ParameterSet &iConfig, edm::ConsumesCollector &cc);
-  
+
+  std::pair<double,double> GetCorrections(const reco::SuperCluster &sc) const;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.